app/views/admin/organizations/index.html.slim

```diff
@@ -3,16 +3,19 @@ div.row
     section.panel
       header.panel-heading.organization-container
         h4 Organizations
+        = link_to "New Organization", new_admin_organization_path, class: "btn btn-primary"
```

Comment:
Do we need i18n here?

Reply:
I once argued that we should make our admin panel use i18n but I think since it's only internal and a few people have access then we don't have to worry about it.